tst/CTA.WebForms2Blazor.Tests/Factories/FileConverterFactoryTests.cs

```diff
@@ -5,13 +5,14 @@
 using NUnit.Framework;
 using CTA.WebForms2Blazor.Factories;
 using CTA.WebForms2Blazor.Services;
+using CTA.WebForms2Blazor.FileConverters;
 
 namespace CTA.WebForms2Blazor.Tests.Factories
 {
     [TestFixture]
-    class FileInformationFactoryTests
+    class FileConverterFactoryTests
     {
-        private const string TestFilesDirectoryPath = "TestingArea/TestFiles";
+        private const string TestFilesDirectoryPath = "TestingArea\\TestFiles";
```

Comment:
can we use path.combine as Jon suggested before so we can support multiple platforms?

Reply:
Whoops, missed this one